frontend/pages/course-evaluation/[id]/index.tsx

```diff
@@ -1,10 +1,21 @@
+import { useEffect } from 'react';
 import SectionTabs from '@/components/CourseEvaluation/SectionTabs';
 import BodyCard from '@/components/utils/BodyCard';
+import useCourseEvaluation from '@/components/hooks/useCourseEvaluation';
+import usePageTitle from '@/components/hooks/usePageTitle';
+import EvaluationHeader from '@/components/Custom/EvaluationHeader';
 
-const Post = () => (
+const Post = () => {
+  
+  const { courseEvaluation } = useCourseEvaluation();
+ 
+  usePageTitle(courseEvaluation.unit_code + " Manage");
```

Comment:
i think its better to use a string template syntax... makes it clear that it is a concatenation